task.go

```diff
@@ -62,6 +64,12 @@ func (e *Executor) Run(ctx context.Context, calls ...taskfile.Call) error {
 		}
 	}
 
+	if e.Summary {
+		firstTask := calls[0].Task
```

Comment:
Hey @jaedle,

The way it's implemented now, the `--summary` flag allows multiple tasks to be specified, but it'll only print the summary for the first one. E.g. `task --summary foo bar` will only print summary for `foo`.

IMHO we should either disallow more than one non-flag arguments or print the summary for all the specified tasks, with a two-empty-line break between then. I prefer the latter.

Reply:
Hey @andreynering,

I completly agree on having multiple summaries. I‘ll change this.